activerecord/lib/active_record/migration.rb

```diff
@@ -622,10 +622,12 @@ def announce(message)
       write "== %s %s" % [text, "=" * length]
     end
 
-    def say(message, subitem=false)
+    def say(message, subitem=false) # :nodoc:
       write "#{subitem ? "   ->" : "--"} #{message}"
     end
 
+    # Prints customized +message+ to the console along with benchmarks
+    # describing how long each step in migration took.
```

Comment:
I'm not sure this is a terribly accurate description of what this method does :confused: